modules/porous_flow/src/userobjects/PorousFlowBrineCO2.C

```diff
@@ -842,6 +885,73 @@ PorousFlowBrineCO2::totalMassFraction(Real pressure,
   return z;
 }
 
+void
+PorousFlowBrineCO2::henryConstant(
+    Real temperature, Real xnacl, Real & Kh, Real & dKh_dT, Real & dKh_dx) const
+{
+  // Henry's constant for dissolution in water
+  Real Kh_h2o, dKh_h2o_dT;
+  _co2_fp.henryConstant_dT(temperature, Kh_h2o, dKh_h2o_dT);
+
+  // The correction to salt is obtained through the salting out coefficient
+  const std::vector<Real> b{1.19784e-1, -7.17823e-4, 4.93854e-6, -1.03826e-8, 1.08233e-11};
```

Comment:
minor: consider std::array for things like this (stack versus heap allocation can help with caching)

Reply:
I always forget this one! Thanks for picking it up - I'll sneak it into #11359

Reply:
And this has prompted me that I swapped over a lot of the fixed vectors in the fluid properties module to arrays as suggested in #11051, but forgot to actually submit a PR. I'll do it now